app/models/liquid_layout.rb

```diff
@@ -2,7 +2,15 @@ class LiquidLayout < ActiveRecord::Base
   has_many :pages
   validates :title, presence: true, allow_blank: false
   validates :content, presence: true, allow_blank: false
-  validate :no_unknown_partials
+
+  # Removing this validation... causing more trouble than it's worth.
+  # There are no hard consequences if an unkown partial is added
+  # to a liquid layout.
+  #
+  # A possible alternative is to provide a warning message when
+  # a partial name is unknown.
+  # 
+  # validate :no_unknown_partials
```

Comment:
I'm really not sold on this. If it's causing trouble, why not make the seeder add partials first and layouts second? The layouts all go in one directory - add them last.

Though there are no catastrophic consequences to adding an unknown partial, it just fails quietly. I've had a lot of my time wasted in the last two weeks by quiet failures, and I don't want us to do it in our code.

The use case I think is pretty clear - someone mistypes a partial name, and instead of quietly letting them add an include that does nothing, the system tells them it's not going to work. That's not part of our workflow now, but it is a part of the expected workflow for power users. I don't think we should compromise their UX when our automated tool (the seeder) can pick up the slack with a little extra logic.